pr-data.csv

```diff
@@ -2284,8 +2284,18 @@ https://github.com/doanduyhai/Achilles,e3099bdce342910951c4862c78751fd81ed4552e,
 https://github.com/doanduyhai/Achilles,e3099bdce342910951c4862c78751fd81ed4552e,integration-test-3_7,info.archinnov.achilles.it.TestEntityWithSASIIndices.should_search_using_like,NOD,,,
 https://github.com/doanduyhai/Achilles,e3099bdce342910951c4862c78751fd81ed4552e,integration-test-3_7,info.archinnov.achilles.it.TestEntityWithSASIIndices.should_search_using_like_non_tokenizer,NOD,,,
 https://github.com/doanduyhai/Achilles,e3099bdce342910951c4862c78751fd81ed4552e,integration-test-3_7,info.archinnov.achilles.it.TestEntityWithSASIIndices.should_search_using_prefix_non_tokenizer,NOD,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.AnnotationUtilTest.getAnnotationSyncAlias,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.AnnotationUtilTest.getAnnotationValueTest,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.AnnotationUtilTest.getAnnotationValueTest2,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.AnnotationUtilTest.getCombinationAnnotationsTest,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.AnnotationUtilTest.getCombinationAnnotationsWithClassTest,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.AnnotationUtilTest.scanMetaAnnotationTest,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.annotation.GenericSynthesizedAggregateAnnotationTest.baseSynthesisAnnotationWorkTest,ID,,,
 https://github.com/dromara/hutool,12969f248329f81f76eb5bece8c718ac1c63f7b1,hutool-core,cn.hutool.core.text.StrJoinerTest.joinMultiArrayTest,OD,Accepted,https://github.com/dromara/hutool/pull/1883,
 https://github.com/dromara/hutool,12969f248329f81f76eb5bece8c718ac1c63f7b1,hutool-core,cn.hutool.core.util.JAXBUtilTest.beanToXmlTest,OD,Accepted,https://github.com/dromara/hutool/pull/1851,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-core,cn.hutool.core.util.ReflectUtilTest.getMethodsFromInterfaceTest,ID,,,
+https://github.com/dromara/hutool,ec14f8384ad384fb6e054df722fe089d64af0c41,hutool-json,cn.hutool.json.Issues1881Test.parseTest,ID,DeveloperWontFix,,
```

Comment:
Why is this `DeveloperWontFix`?

Reply:
https://github.com/dromara/hutool/pull/2713

Reply:
The final comment in English:
Fine, leave it be until there're problems. The test has no serious problem.
BTW, I think you should also include the PR Link in the two rows